scripts/installer/windows/scripts/pf.cmd

```diff
@@ -0,0 +1,5 @@
+@echo off
+setlocal
+
+set MAIN_EXE=%~dp0.\main.exe
```

Comment:
maybe give this exe some other name more identifiable. better with pf prefix. e.g. pfcli?

Reply:
change main to pfcli